docker-compose.yml

```
@@ -0,0 +1,88 @@
+version: '2.3'
+
+x-volumes:
+  &default-volumes
+    volumes:
+      - files:/app/web/sites/default/files
+
+x-environment:
+  &default-environment
+    LAGOON_ROUTE: ${LOCALDEV_URL:-http://govcms8.docker.amazee.io}
+
+volumes:
+  files: {}
+
+services:
+
+  cli: # cli container, will be used for executing composer and any local commands (drush, drupal, etc.)
+    build:
+      context: .
+      dockerfile: $PWD/.docker/Dockerfile.govcms8
+    image: ${DOCKERHUB_NAMESPACE:-govcms}/govcms8
```

```suggestion
    image: ${DOCKERHUB_NAMESPACE:-govcms8}/govcms8
```
places in correct default docker hub namespace